addons/event/report/event_event_templates.xml

```diff
@@ -1,104 +1,94 @@
+                                    <span itemprop="startDate" t-field="event.date_begin" t-options="{&quot;hide_seconds&quot;: True}"/>
                                 </div>
-                                <div class="row">
-                                    <div t-field="o.event_id.badge_front"></div>
+                                <div>
+                                    <strong>To:</strong>
+                                    <span itemprop="endDate" t-field="event.date_end" t-options="{&quot;hide_seconds&quot;: True}"></span>
                                 </div>
                             </div>
                         </div>
-                        <!-- Back -->
-                        <div class="col-xs-6" style="padding-right:7mm; height: 149mm; -webkit-transform:rotate(180deg); padding-top: 2mm;">
-                            <span t-field="o.event_id.badge_back"/>
+                        <div class="row mt32">
+                            <div class="col-xs-offset-1 col-xs-1 mt32">
+                                <i class="fa fa-user fa-3x" aria-hidden="true"/>
+                            </div>
+                            <t t-if="attendee">
+                                <div class="col-xs-4 mt32" id="o_event_attendee_name">
+                                    <h3 t-field="attendee.name" class="mb0 mt0"/>
+                                </div>
+                            </t>
+                            <t t-else="not attendee">
+                                <div class="col-xs-4 mt32" id="o_event_attendee_name_edit">
+                                    <h3 class="mb0 mt0">Attendee Name</h3>
+                                </div>
+                            </t>
                         </div>
                     </div>
-                    <div class="row">
-                        <!-- Inner left -->
-                        <div class="col-xs-6 text-center" style="padding-right:7mm; border-top:1px dashed black; height: 148mm;">
-                            <span t-field="o.event_id.badge_innerleft"/>
-                        </div>
-                        <!-- Inner right -->
-                        <div class="col-xs-6 text-center" style="border-left:1px dashed black; border-top:1px dashed black; height: 148mm; text-center">
-                            <span t-field="o.event_id.badge_innerright"/>
+                    <div class="panel-footer">
+                        <div t-field="event.badge_footer">
+                            <div class="row">
+                                <div class="col-xs-offset-2 col-xs-10">
+                                    <strong> Terms and conditions </strong>
+                                    <ul>
+                                        <li>Please print this ticket in A4 page and bring with you to the event.</li>
+                                        <li>We expect participants to come 30 minutes before event.</li>
```

Comment:
"the" is missing before event : <li>We expect participants to come 30 minutes before the event.</li>